lib/commands/serve.js

```diff
@@ -41,8 +41,10 @@ module.exports = Command.extend({
     return this._checkExpressPort(commandOptions)
       .then(this._autoFindLiveReloadPort.bind(this))
       .then(function(commandOptions) {
+        var config = this.project.config(commandOptions.environment);
+
         commandOptions = assign({}, commandOptions, {
-          baseURL: this.project.config(commandOptions.environment).baseURL || '/'
+          baseURL: config.rootURL || config.baseURL || '/'
```

Comment:
Because `baseURL` will be deprecated, and I'd rather not have to think "is this referencing the old deprecated thing or the coerced thing" everytime I see the word `baseURL` in the repo.